app/resources/forms.js.template

```diff
@@ -375,4 +375,14 @@ function make_form_expandable(expandedSection) {
   });
 }
 
+// Show the side menu
+function menuOpenNav() {
```

Comment:
Generally the function name should be a verb. e.g., showSideMenu() would be a better name. Same for below.